tests/flamegraph.rs

```diff
@@ -52,7 +52,7 @@ fn test_flamegraph_multiple_files(
 
     let expected_len = metadata.len() as usize;
     let mut result = Cursor::new(Vec::with_capacity(expected_len));
-    flamegraph::from_files(&mut options, &input_files, &mut result)?;
+    flamegraph::from_files(&mut options, &input_files, &mut result).unwrap();
```

Comment:
I think the test failure are due to this change.

Reply:
You're right. Reverted.